admin/app/notify_client/service_api_client.py

```diff
@@ -494,5 +494,15 @@ def update_service_data_retention(self, service_id, data_retention_id, days_of_r
     def get_service_data_retention(self, service_id):
         return self.get("/service/{}/data-retention".format(service_id))
 
+    def get_failing_callback_stats(self):
```

Comment:
thoughts on putting this into a new API client file that refers to the domain of our new model thats being exposed -- `notify_client/callback_failures_client.py` ?

Reply:
Sounds good to me!